src/components/Registration/BusinessAddresses.vue

```diff
@@ -293,76 +293,33 @@ export default class BusinessAddresses extends Mixins(CommonMixin) {
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
 
-.meta-container {
```

Comment:
I replaced all this flex stuff (which isn't very responsive) with rows and cols.